src/app/page/Home/Home.js

```diff
@@ -4,62 +4,141 @@ import PropTypes from "prop-types";
+import { Grid, LinearProgress } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
 
-import { BigLogo, MissionText, StyledHomeButton } from "./Home.style";
+import { StyledHomeButton } from "./Home.style";
+import HomeImage1 from "../../../img/HomeImage1.png";
+import HomeImage2 from "../../../img/HomeImage2.png";
+import SplashImage1 from "../../../img/SplashImage1.png";
+
+const useStyles = makeStyles((theme) => ({
+  HomeImage: {
+    width: "100%",
+  },
+  LoginButton: {
+    width: "200px",
+    height: "48px",
+    margin: "24px auto",
+  },
+  SignupButton: {
+    width: "200px",
+    height: "48px",
+    margin: "12px auto",
+  },
+  Paragraph: {
+    paddingBottom: "12px",
+    textAlign: "left",
+  },
+
+  LoadingScreenContainer: {
+    width: "inherit",
+    overflowX: "hidden",
+  },
+  SplashImage: {
+    height: "calc(100vh - 98px)",
+  },
+  SplashTitle: {
+    position: "absolute",
+    top: "30vh",
+    left: "0",
+    right: "0",
+    margin: "auto",
+    color: "white",
+  },
+  SplashSub: {
+    position: "absolute",
+    top: "40vh",
+    left: "0",
+    right: "0",
+    margin: "auto",
+    color: "white",
+    fontSize: "24px",
+  },
+}));
+
+const LoadingComponent = () => {
+  const classes = useStyles();
+  return (
+    <Grid className={classes.LoadingScreenContainer}>
+      <img src={SplashImage1} className={classes.SplashImage} alt="" />
+      <H1 className={classes.SplashTitle}>Resilience</H1>
+      <H3 className={classes.SplashSub}>Neighbors helping neighbors</H3>
+    </Grid>
+  );
+};
 
 const HomePage = ({ history, ...rest }) => {
-  const missions = useSelector((state) => state.firestore.ordered.missionsVolunteered);
+  const classes = useStyles();
   const isEmpty = useSelector((state) => state.firebase.auth.isEmpty);
   const isLoaded = useSelector((state) => state.firebase.auth.isLoaded);
 
   return (
-    <Page isLoaded={isLoaded}>
+    <Page isLoaded={isLoaded} LoadingComponent={LoadingComponent}>
       {isEmpty ? (
-        <>
-          <BigLogo>
-            <Logo />
-          </BigLogo>
-          <MissionText>Global Community, Local Mutual Aid</MissionText>
-          <StyledHomeButton onClick={() => history.push("/login")}>Sign In/Up</StyledHomeButton>
-          <StyledHomeButton onClick={() => history.push("/missions")}>
-            View Missions
-          </StyledHomeButton>
-          <StyledHomeButton
-            onClick={() => history.push("/request/create")}
-            text="Request Help"
-            secondary
-          />
-          <StyledHomeButton onClick={() => history.push("/about")}>About</StyledHomeButton>
-        </>
+        <Grid container>
+          <Grid container>
+            <img src={HomeImage1} className={classes.HomeImage} alt="" />
+          </Grid>
+          <Grid container>
+            <Button className={classes.LoginButton} onClick={() => history.push("/login")}>
+              Log in
```

Comment:
Can we rename to Signin instead of login?

Reply:
yeah